.github/workflows/ci.yml

```diff
@@ -6,8 +6,8 @@ on:
       - "wip/**"
       - "main"
   pull_request_target:
-    branches-ignore:
```

Comment:
Turns out branches for PR events are the target branches, e.g. `main` and `branches-ignore` basically doesn't make any sense here as it would match all PRs that don't target `/actions/*`